styles/styles.scss

```diff
@@ -275,6 +275,12 @@ p {
   }
 }
 
+.btn-red {
```

Comment:
btn-danger exists, as well as border border-danger and color-danger

https://react-bootstrap.github.io/components/buttons/
https://getbootstrap.com/docs/4.0/utilities/borders/#border-color
https://getbootstrap.com/docs/4.0/utilities/colors/